httpgrpc/server/server.go

```diff
@@ -11,13 +11,12 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/grpc-ecosystem/go-grpc-middleware"
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
```

Comment:
Change with no explanation in description.

Reply:
It was probably my code editor trying to be smart. I suppose this is because the package name does not match the import path. Anyway, I reverted this change.